### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by timo-piirmann**

### CODE REVIEW

File: grails-app/views/_download.gsp [hunk 1]
```diff
@@ -80,56 +70,56 @@
...
                     </p>
                 </g:if>
 
                 <div id="statusMsg" style="text-align: center; font-weight: bold; "></div>
 
                 <script type="text/javascript">
-                    $(document).ready(function() {
-                        // catch download submit button
-                        // Note the unbind().bind() syntax - due to Jquery ready being inside <body> tag.
+                    $(document).ready(function () {
```
Comment (filippivanov) [inline grails-app/views/_download.gsp]: We don't really have the style guide for ala, but plutof uses (used before es6) ``function(`` . It's not tabs-vs-spaces, but very close :)

File: grails-app/views/_speciesList.gsp [hunk 2]
```diff
@@ -77,65 +77,53 @@
                         ${list.listType?.getDisplayValue()}
                     </td>
 
-                    <g:if test="${request.isUserInRole("ROLE_ADMIN")}">
+                    <g:if test="${request.isUserInRole(" ROLE_ADMIN")}">
```
Comment (filippivanov) [inline grails-app/views/_speciesList.gsp]: Is it supposed to be changed? Not sure how ``isUserInRole`` works, but doubtful it trims the input

Comment (timo-piirmann) [inline grails-app/views/_speciesList.gsp]: I did some default styling with my editor, which doesn't seem to handle well " inside "" in grails tags

File: grails-app/views/layouts/generic.gsp [hunk 3]
```diff
@@ -2,33 +2,33 @@
 <html lang="en">
     <head>
         <meta charset="utf-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <g:render template="/manifest" plugin="elurikkus-commons" />
+            <meta http-equiv="X-UA-Compatible" content="IE=edge">
+                <meta name="viewport" content="width=device-width, initial-scale=1">
+                    <g:render template="/manifest" plugin="elurikkus-commons"/>
```
Comment (filippivanov) [inline grails-app/views/layouts/generic.gsp]: This looks auto-indented. <meta> tags are not closed in html

File: grails-app/views/public/specieslists.gsp [hunk 4]
```diff
@@ -1,103 +1,98 @@
...
-  - the License at http://www.mozilla.org/MPL/
-  -
-  - Software distributed under the License is distributed on an "AS
-  - IS" basis, WITHOUT WARRANTY OF ANY KIND, either express or
-  - implied. See the License for the specific language governing
-  - rights and limitations under the License.
-  --}%
+%{-- - Copyright (C) 2012 Atlas of Living Australia - All Rights Reserved. - - The contents of this file are subject to the Mozilla Public - License Version 1.1 (the "License"); you may not use this file - except in compliance with the License. You
+may obtain a copy of - the License at http://www.mozilla.org/MPL/ - - Software distributed under the License is distributed on an "AS - IS" basis, WITHOUT WARRANTY OF ANY KIND, either express or - implied. See the License for the specific language
+governing - rights and limitations under the License. --}%
```
Comment (filippivanov) [inline grails-app/views/public/specieslists.gsp]: Also mangled

File: grails-app/views/speciesList/upload.gsp [hunk 5]
```diff
@@ -1,606 +1,568 @@
...
-                        <input
-                            type="button"
-                            id="uploadButton"
-                            class="datasetName actionButton erk-button erk-button--light"
-                            value="Upload"
-                            onclick="javascript:uploadSpeciesList();" />
+                                </tbody>
+                            </table>
+
+                            <input type="button" id="uploadButton" class="datasetName actionButton erk-button erk-button--light" value="Upload" onclick="javascript:uploadSpeciesList();"/>
```
Comment (filippivanov) [inline grails-app/views/speciesList/upload.gsp]: Multi-attribute elements also are more readable expanded:
```
   <input
        type="button"
        id="uploadButton"
    />
```

**Pull request merged**